src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/EELocalSymbol.cs

```diff
@@ -51,7 +53,7 @@ internal sealed class EELocalSymbol : EELocalSymbolBase
             bool isCompilerGenerated,
             bool canScheduleToStack)
         {
-            Debug.Assert(method != null);
+            RoslynDebug.Assert(method != null);
             Debug.Assert(ordinal >= -1);
             Debug.Assert(!locations.IsDefault);
             Debug.Assert((object)type != null);
```

Comment:
>Debug.Assert((object)type != null); [](start = 12, length = 35)

I'm surprised this assertion doesn't cause a W warning below on `_type = type;`. How come?